docs/changelog.rst

```diff
@@ -9,6 +9,7 @@ Develop
 
 * [BUGFIX] Allow decimals without leading zero in evaluation parameter URN
 * [ENHANCEMENT] Enable instantiation of a validator with a multiple batch BatchRequest
+* [ENHANCEMENT] Adds a batch_request_list parameter to DataContext.get_validator to enable instantiation of a Validator with batches from multiple BatchRequests
```

Comment:
@talagluck @anthonyburdi -- Just a comment -- we will have to update `ChangeLog` judiciously (as in, not write everything we did in the GE-2 Epic, but still include salient points) prior to issuing a PR into `develop`.  Thanks.